frontend/src/styles/variables.scss

```diff
@@ -0,0 +1,34 @@
+$box-padding: 25px;
```

Comment:
If you decided to use scss for styling app, please use the most features of it.
Set up variables and typography files.
in typography set 
`
html {   font-size: 10px; }
`
end then use rem values everywhere in the styles to set up margins, paddings etc.